src/main/webapp/style.css

```diff
@@ -222,4 +222,14 @@ body {}
   color: rgb(205, 236, 232);
   font-size: 100px;
   text-align: center;
-}
\ No newline at end of file
+}
+
+.taskComplete {
+  border: thin solid black;
+  background-color: greenyellow;
+  display: flex;
+  margin: 10px;
+  padding: 10px;
+  width: 300px;
+}
+ 
```

Comment:
Why does github say there is no newline at the end of the file?

Reply:
not sure

Reply:
no sure